<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnJugador</class>
 <widget class="QWidget" name="UnJugador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="boton_pedir_carta">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>540</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Pedir Carta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_retirarse">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Retirarse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_doblar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>540</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Doblar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_dividir">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>540</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Dividir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saldo</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pantalla">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="selector_apuesta">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>440</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce una apuesta inicial: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_empezar">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>420</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Empezar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_valor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_palo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Palo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_retirada">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>90</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Palo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>310</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Valor de la mano:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pantalla_2">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>320</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pantalla_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Valor de la banca:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
